deploy/clowdapp.yaml

```diff
@@ -2646,7 +2646,7 @@ parameters:
 - displayName: Worker Queue
   name: SCHEDULER_WORKER_QUEUE
   required: true
-  value: cost_model,download,ocp,priority,refresh,summary
+  value: cost_model,download,hcs,ocp,priority,refresh,summary
```

Comment:
You may want to add this worker queue name to our local docker-compose yaml as well. Totally optional though because it will just get picked up by the default celery worker. 🤷

Reply:
Done